package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
The main/module entries should take care of this.  Those are the entry points in dist/.  The src/ will keep backwards compat until we start importing in other modules without src/